app/views/account/index.html

```diff
@@ -8,7 +8,7 @@
 
 {% block formContent %}
 
-  <h1 class="govuk-heading-l">Create a GOV.UK account or sign in</h1>
+  <h1 class="govuk-heading-l">Create a GOV.UK One Login account or sign in</h1>
```

Comment:
Should 'One Login' be lowercase?

Reply:
Just had an email from Joe with confirmation of the name and this is correct